python/cudf/cudf/utils/utils.py

```diff
@@ -622,3 +623,18 @@ def _categorical_scalar_broadcast_to(cat_scalar, size):
         offset=codes.offset,
         ordered=ordered,
     )
+
+
+def create_pandas_series(
```

Comment:
This name is kinda ambiguous, any ideas on a more descriptive name? Also could you add a brief docstring?

Reply:
Sure will add docstring, but should we name it `back_ward_compatible_pandas_series`(looks lengthy). or `create_pandas_series_handle_empty_data_dtype` ( This one is too lengthy than previous one 😅 )?

Reply:
I think we can leave the name but make it an internal function and then add a brief docstring and we should be fine.

Reply:
Done 👍